### ISSUE TITLE
Improve "else if" support

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#151]
Currently, when you cascade a lot of `else if` statements, it gets translated in a nested fashion. Example:

```
		NSUInteger group = 1;
		if ([self numberOfYears] > 3 && [self numberOfYears] <= 5){
			group = 2;
		}else if ([self numberOfYears] > 5 && [self numberOfYears] <= 10){
			group = 3;
		}else if ([self numberOfYears] > 10 && [self numberOfYears] <= 20){
			group = 4;
		}else if ([self numberOfYears] > 20 && [self numberOfYears] <= 30){
			group = 5;
		}else if ([self numberOfYears] > 30){
			group = 6;
		}
```

Current iSwift output:

```
			if self.numberOfYears > 3&&self.numberOfYears <= 5 {
				group = 2
			} else {
				if self.numberOfYears > 5&&self.numberOfYears <= 10 {
					group = 3
				} else {
					if self.numberOfYears > 10&&self.numberOfYears <= 20 {
						group = 4
					} else {
						if self.numberOfYears > 20&&self.numberOfYears <= 30 {
							group = 5
						} else {
							if self.numberOfYears > 30 {
								group = 6
							}
							
						}
						
					}
					
				}
				
			}
```

As you can see from a fairly simple few lines of code, you get a mayhem of nested `if`s...

**Comment by drkameleon** [comment]
Also related (now that I've noticed lol): 

https://github.com/drkameleon/iswift/issues/165

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.4. That was very tricky to resolve. But a superb catch from your side. (I really have no idea how I have missed it...). Thanks a lot - one more time! :)

---

**Example:**

```objective-c
NSUInteger group = 1;
if ([self numberOfYears] > 3 && [self numberOfYears] <= 5){
    group = 2;
}else if ([self numberOfYears] > 5 && [self numberOfYears] <= 10){
    group = 3;
}else if ([self numberOfYears] > 10 && [self numberOfYears] <= 20){
    group = 4;
}else if ([self numberOfYears] > 20 && [self numberOfYears] <= 30){
    group = 5;
}else if ([self numberOfYears] > 30){
    group = 6;
}
```

**Output:**

```swift
var group: UInt = 1
if self.numberOfYears() > 3 && self.numberOfYears() <= 5 {
	group = 2
} else if self.numberOfYears() > 5 && self.numberOfYears() <= 10 {
	group = 3
} else if self.numberOfYears() > 10 && self.numberOfYears() <= 20 {
	group = 4
} else if self.numberOfYears() > 20 && self.numberOfYears() <= 30 {
	group = 5
} else if self.numberOfYears() > 30 {
	group = 6
}
```
